suites/reef/rados/test_rados_all_generic_features.yaml

```diff
@@ -656,7 +656,7 @@ tests:
           plugin: clay
           app_name: rbd
           erasure_code_use_overwrites: "true"
-          crush-failure-domain: osd
```

Comment:
Let's modify the k,m,d values to come in the range of no of hosts with the change in failure domain

Reply:
Current values of k,m, and d will work, shall we increase them?